config/tracks/solitude1964.ini

```diff
@@ -1,62 +1,578 @@
+[ABOUT]
+AUTHOR = leBluem, Sliderman
+VERSION = 1.01
+DATE_RELEASE = 2019 jan 4, 01.07.19, 03.12.19, 06.08.2021
+LIGHTS_COUNT = 
+NOTES = solitude1964 tweaked meshes, materials and lights, grass, water, rain, wind
+
+[BASIC]
+SUPPORTS_WIND = 1
+
+[WIND]
+;IGNORE =
+;DYNAMIC_FLAGS = AUTO
+
 [LIGHTING]
-LIT_MULT = 1
+LIT_MULT = 1.0
+CAR_LIGHTS_LIT_MULT = 1.0
+SPECULAR_MULT = 1.0
+
+[MODEL_REPLACEMENT_0]
+ACTIVE = 1
+DESCRIPTION = hide some objects like crowd
+
+[INCLUDE]
+INCLUDE = common\conditions.ini, common\materials_base.ini, common\materials_track.ini, common\conditions_seasons.ini
+
+[CONDITION_4]
+INPUT = wfx_WET
+NAME = WET
+LUT = (|-1=0|0=0|1=1|)
+
+[CONDITION_95]
+NAME = SEASON_SUMMER_2
+INPUT = YEAR_PROGRESS
+LUT = (|-1=0|0=0|0.45=0|0.6=0.85|0.7=0|1=0|)
+LAG = 0
+
+[CONDITION_96]
+NAME = SEASON_MUD
+INPUT = YEAR_PROGRESS
+LUT = (|-1=0|0=0|0.1=0|0.2=0.85|0.3=0|0.7=0|0.8=0.85|0.9=0|1=0|)
+LAG = 0
+
+[CONDITION_97]
+NAME = SEASON_SUMMER
+INPUT = YEAR_PROGRESS
+LUT = (|-1=0|0=0|0.1=0|0.2=1|0.3=0|0.45=0|0.6=0.85|0.7=0|0.71=0|0.8=1|0.9=0|1=0|)
+LAG = 0
+
+[CONDITION_98]
+NAME = SEASON_WINTER
+INPUT = YEAR_PROGRESS
+LUT = (|-1=0|0=0.75|0.075=1|0.2=0|0.8=0|0.9=0.5|1=0.75|)
+LAG = 0
+
+[CONDITION_99]
+NAME = SEASON_AUTUMN
+INPUT = YEAR_PROGRESS
+LUT = (|-1=0|0=0.75|0.2=0|0.45=0|0.6=0.175|0.7=0|0.75=0|0.8=0.75|1=0.75|)
+LAG = 0
+
+[Material_Water]
+Materials = grvl_water
+Type = LAKE
+UseNormalMapTwice = 0
+UseAlphaFromDiffuse = 1
+UseRecommendedTextures = 1
+
+[MATERIAL_ADJUSTMENT_1]
+ACTIVE = 1
+DESCRIPTION = horizon a bit darker
+MATERIALS = bk?
+KEY_0 = ksAmbient
+VALUE_0 = 0.175
+KEY_1 = ksDiffuse
+VALUE_1 = 0.12
+
+[MATERIAL_ADJUSTMENT_2]
+ACTIVE = 1
+DESCRIPTION = tarmac a bit darker
+MATERIALS = road0?
+KEY_0 = ksAmbient
+VALUE_0 = 0.4125
+KEY_1 = ksDiffuse
+VALUE_1 = 0.5125
```

Comment:
0.4 and 0.5 diffuse/ambient respectively will cause issues with most mods, consider sticking to values within the 0.2 - 0.35 range.